<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGamepadOptionsWidgetBase</class>
 <widget class="QWidget" name="QgsGamepadOptionsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>792</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GPS Device Editor</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mDeviceCombo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>611</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Left axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLeftAxisLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mRightAxisLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Right axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Right axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Button A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Button B</string>
   </property>
  </widget>
  <widget class="QLabel" name="mButtonALabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mButtonBLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mButtonXLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Button X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Button Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="mButtonYLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Center button</string>
   </property>
  </widget>
  <widget class="QLabel" name="mCenterButtonLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLeftButtonLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>560</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QLabel" name="mRightButtonLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>560</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="mUpButtonLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>510</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mDownButtonLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>630</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>630</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QLabel" name="mGuideButtonLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Guide button</string>
   </property>
  </widget>
  <widget class="QLabel" name="mStartButtonLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Start button</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Select button</string>
   </property>
  </widget>
  <widget class="QLabel" name="mSelectButtonLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Left 1 trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Left 2 trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Right 1 trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Right 2 trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelButtonL1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelButtonL2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelButtonR1">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>320</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelButtonR2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelButtonL3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Left 3 button</string>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelButtonR3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Right 3 button</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
